cli/flox-rust-sdk/src/flox.rs

```diff
@@ -106,6 +106,8 @@ pub struct Features {
     pub upload: bool,
     #[serde(default)]
     pub compose: bool,
+    #[serde(default)]
+    pub qa: bool,
 }
```

Comment:
I would rather make the field private and add accessor methods instead that are basically `self.<the field> || self.qa`.
Modifying the config, even if in memory has the effect that any `flox config` command (or automatic login to floxhub) will actually write the modified config values e.g. `features.compose = true` if `features.qa` was set, which makes enabling features easy but disabling them not.

Reply:
Ah good catch. Fixed